SDPi_Supplement/asciidoc/volume2/dev-24/tf2-ch-a-mdpws-dev-24.adoc

```diff
@@ -39,6 +39,35 @@ include::../../listings/vol2-clause-appendix-a-mdpws-dev-24-probe.xml[]
 `s12:Envelope/s12:Body/wsd:Probe/wsa:Types`:: List that contains at least `mdpws:MedicalDevice` to express seeking <<actor_somds_provider>>s that conform to MDPWS.
 `s12:Envelope/s12:Body/wsd:Hello/wsd:Scopes/@MatchBy`:: The algorithm used to compare the `s12:Envelope/s12:Body/wsd:Hello/wsd:Scopes` against the <<actor_somds_provider>>'s scopes.
 `s12:Envelope/s12:Body/wsd:Hello/wsd:Scopes`:: The <<payload_dev_24_probe_discovery_scope>> as a list of URIs to probe for.
+[#vol2_clause_appendix_mdpws_dev_24_recurring_probe]
+====== Recurring Probe
+While the number of Hello messages scales linearly with the number of <<actor_somds_provider>>s, in the worst case the number of Probe / ProbeMatch messages is the product of the number of <<actor_somds_provider>>s and the number of <<actor_somds_consumer>>s. Therefore, in order to prevent the network congestion from Probe / ProbeMatch messages, the following requirements are defined:

```

Comment:
```suggestion
While the number of Hello messages scales linearly with the number of <<actor_somds_provider>>s, in the worst case the number of Probe / ProbeMatch messages is the product of the number of <<actor_somds_provider>>s and the number of <<actor_somds_consumer>>s.
Therefore, in order to prevent the network congestion from Probe / ProbeMatch messages, the following requirements are defined:

```